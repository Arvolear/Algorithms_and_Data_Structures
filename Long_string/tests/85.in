4630883258154764850535287720735110209746573159
3759203102500971095282769601118586618948110109