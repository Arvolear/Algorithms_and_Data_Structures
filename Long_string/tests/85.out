8390086360655735945818057321853696828694683268
871680155653793755252518119616523590798463050
17408430711375197480177253198625274087750580479983270688432189451171383208105222602155964331
1